integration/cmd/search-adhoc-request/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+
+	"github.com/Telenav/osrm-backend/integration/pkg/api/search/coordinate"
+	"github.com/Telenav/osrm-backend/integration/pkg/api/search/nearbychargestation"
+	"github.com/Telenav/osrm-backend/integration/pkg/backend"
+)
+
+func main() {
+	flag.Parse()
+
+	if len(flags.entityEndpoint) == 0 {
+		fmt.Println("[ERROR] -entity should not be empty!")
+		return
+	}
+
+	if len(flags.apiKey) == 0 {
+		fmt.Println("[ERROR] -apikey should not be empty.")
+		return
+	}
+	if len(flags.apiSignature) == 0 {
+		fmt.Println("[ERROR] -apisignature should not be empty.")
```

Comment:
Normally I'll suggest we always use `glog` for log output. In contrast to `stdout` of `fmt.Print`, it's default target is `stderr`. It can help us for unified log format(always with timestamp, log level, etc.). 
Only use `fmt.Print` on contents that possible to used by downstream tool. It can helpful for command tool combination.